ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -17,24 +19,40 @@ export class PrepareMigrationComponent implements OnInit {
   displayedColumns = ['Title', 'Source', 'Destination']
   dataSource: any = []
   migrationModes: any = []
+  subscription!: Subscription
   constructor(
     private dialog: MatDialog,
     private fetch: FetchService,
     private snack: SnackbarService,
-    private targetDetailService: TargetDetailsService
-  ) {}
+    private data: DataService
+  ) { }
 
-  isTargetDetailSet: boolean = false
-  isStreamingCfgSet: boolean = false
-  isSchemaMigration: boolean = true
   isStreamingSupported: boolean = false
-  selectedMigrationMode: string = MigrationModes.schemaOnly
-  selectedMigrationType: string = 'bulk'
-  targetDetails: ITargetDetails = this.targetDetailService.getTargetDetails()
+  selectedMigrationMode: string = 'Schema'
```

Comment:
should use a constant / enum if possible

Reply:
done